src/02-fullfill-a-promise.js

```diff
@@ -0,0 +1,9 @@
+require('es6-promise');
+
+const promise = new Promise(function(fulfill, reject) {
```

Comment:
The other files use `fulfilled` as the parameter, so this one should be "fulfilled" instead of "fulfill" for consistency